amy/workshops/templatetags/training_progress.py

```diff
@@ -7,19 +9,21 @@
 register = template.Library()
 
 
-@register.simple_tag
-def progress_label(progress):
-    assert isinstance(progress, TrainingProgress)
-
+def progress_state_class(state):
     switch = {
         "n": "warning",
         "f": "danger",
         "a": "info",
         "p": "success",
     }
-    additional_label = switch[progress.state]
+    return switch[state]
+
 
-    fmt = "badge badge-{}".format(additional_label)
+@register.simple_tag
+def progress_label(progress):
+    assert isinstance(progress, TrainingProgress)
```

Comment:
I think you can get rid of this assertion. Use type annotation for `progress` argument instead.